packages/kbn-cases-components/src/tooltip/skeleton.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { EuiFlexItem, EuiLoadingContent } from '@elastic/eui';
+
+const SkeletonComponent: React.FC = () => {
+  return (
+    <EuiFlexItem style={{ width: 240 }} data-test-subj="tooltip-loading-content">
+      <div style={{ width: 70, marginBottom: '12px' }}>
+        <EuiLoadingContent lines={1} />
+      </div>
+
```

Comment:
nit: I am not sure how new lines in JSX are treated by Reac & browsers. Should we remove them?

Reply:
I added it so that the height flicker is not so significant when it changes from` loading state to data state`

https://user-images.githubusercontent.com/117571355/212702493-eb6c2cf3-1fc2-4d81-9ab5-4c6ed690d94b.mov



Verified in `chrome, firefox and Safari` and it works fine. Safari doesn't seem to recognise round border though 😄 
 
<img width="1298" alt="image" src="https://user-images.githubusercontent.com/117571355/212703480-2265ff69-1970-4674-b846-7b6692ad3ca8.png">

Reply:
I see. If you want to add space then you can use [`EuiSpacer`](https://elastic.github.io/eui/#/layout/spacer).